Functions.Templates/Templates/BlobTrigger-Custom/function.json

```diff
@@ -0,0 +1,11 @@
+{
+    "bindings": [
```

Comment:
Please rename folder to include handler BlobTrigger-CustomHandler

Reply:
Thought it would be better to match folder name suffixes to `FUNCTIONS_WORKER_RUNTIME` value as much as possible. I also see that the other folders maybe strives to keep that ?

Reply:
Makes sense. Lets leave them as Custom